cmd/icingadb/main.go

```diff
@@ -211,14 +211,53 @@ func run() int {
 
 						g.Go(func() error {
 							configInitSync.Wait()
+
+							elapsed := time.Since(syncStart)
+							logger := logs.GetChildLogger("config-sync")
+							if synctx.Err() == nil {
+								logger.Infof("Finished config sync in %s", elapsed)
+							} else {
+								logger.Warnf("Aborted config sync after %s", elapsed)
+							}
+
+							return nil
+						})
+
+						g.Go(func() error {
+							stateInitSync.Wait()
+
+							elapsed := time.Since(syncStart)
+							logger := logs.GetChildLogger("config-sync")
+							if synctx.Err() == nil {
+								logger.Infof("Finished inital state sync in %s", elapsed)
```

Comment:
Oh, and while reading the diff, I saw that "initial" is misspelled here.

Reply:
Oh noes. Ty!